### DISCUSSION TITLE
SIDKick Schematic

### DISCUSSION THREAD

**Discussion by guyleslie** [frntc/Sidekick64#38] [General]
Can you share the SIDKick schematics?

**Comment by frntc** [comment]
Sure, give me a few days to prepare that (busy atm). In fact it's using the same principles as the Sidekick to read and write signals.

**Reply by guyleslie** [reply]
Thanks! Your project is unbelievable good. I think you made history in c64 accessories ...

**Reply by frntc** [reply]
Thanks!

The schematics are now in the SIDKick's Gerber-directory.
